CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## Current Master
 - Show "Buy Ether" button and warning on tx confirmation when sender balance is insufficient
 
```

Comment:
Please preserve formatting by having space between headers but not between line items.